docs/adr/XXXX-timezones.md

```diff
@@ -0,0 +1,83 @@
+# timezones
+
+* Status: proposed
+* Deciders: [Sander, Wouter, Arnaud, Bart]
+* Date: 2023-01-09
+
+Technical Story: [
+    https://github.com/inmanta/inmanta-core/issues/2919,
+    https://github.com/inmanta/inmanta-core/pull/2953,
+    https://github.com/inmanta/inmanta-lsm/pull/621,
+]
+
+## Context and Problem Statement
+
+How do we deal with timezones when it comes to datetime data? This data is passed in both directions over the API, stored at
+various places in the database and manipulated throughout the code. Which invariants do we uphold with regards to explicit
+timezone information for these datetime objects?
```

Comment:
I think the problem statement is not sufficiently clear. This document often talks about backwards compatibility issues. So basically we describe here how we went from implementation X to implementation Y regarding the handling of timezones. I think we should clearly describe here what the implementation X was. That will make it easier for the reader to interpret the backwards compatibility constraints discussed later on in the document.

Reply:
I added some more context, I think it should be clear now.